bootstrap/startup.json.default

```diff
@@ -0,0 +1,29 @@
+{
+    "core": {
+        "tag": "master",
```

Comment:
this will work if we update the tag but it's using the old master version ?

Reply:
the only possible issue here is that we push over the same tag and it uses the local one instead of updating. I don't think we should do that, btw

Reply:
[Common, yes](https://medium.com/@mccode/the-misunderstood-docker-tag-latest-af3babfd6375#:~:text=Latest%20is%20just%20a%20tag%20with%20a%20special%20name.&text=%E2%80%9CLatest%E2%80%9D%20simply%20means%20%E2%80%9Cthe,build%20%2Dt%20marc%2Ftest%20.&text=sudo%20docker%20build%20%2Dt%20marc%2Ftest%20.,-sudo%20docker%20tag).
We could use the master tag to check if there are updates and then update it here so it points to the "latest" (the the version tag instead).
I put master in here because we don't have any other tags and I didn't feel like creating a new one for no reason.
I also still prefer the idea of having "curated" lists of versions/tags

Reply:
We discussed this on a call. I'll add a "docker pull" before running, as  that is harmless and will help handling cases of force-pushing tags (useful while in development, I don't endorse it)

Reply:
we can also use the hash instead of tag